cadmiumcat/books-api
# Feature/reviews

### This PR

- Adds swagger spec to allow reviews on books. It adds the spec for the endpoints:
GET `/books/{id}/reviews/{review_id}` 
GET `/books/{id}/reviews`
POST `/books/{id}/reviews`
PUT `/books/{id}/reviews/{review_id}`

- Fixes typos
- Adds swagger spec page https://cadmiumcat.github.io/books-api/

### How to review
- Check that the are no errors in the spec (you can copy and paste into https://editor.swagger.io/
- Check the new endpoints make sense
